src/System.Management.Automation/namespaces/ProviderBase.cs

```diff
@@ -37,7 +39,7 @@ public interface ICmdletProviderSupportsHelp
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Maml", Justification = "Maml is an acronym.")]
         string GetHelpMaml(string helpItemName, string path);
```

Comment:
The documentation for this method does not specify what a `null` return value means. I would expect the _implementations_ and _callers_ of this interface to be annotated at the same time as this interface to ensure the pre- and post-conditions are correctly captured.